redash/query_runner/couchbase.py

```diff
@@ -17,7 +19,7 @@
 
 TYPES_MAP = {
     str: TYPE_STRING,
-    unicode: TYPE_STRING,
+    text_type: TYPE_STRING,
```

Comment:
@arikfr @cclauss I think this and other cases of the `TYPES_MAP` is incorrect, since we're now on Python 3 and `str` is Unicode, while `text_type` is as well. While we were on Python 2 that wasn't the case. I'll send a PR.

Reply:
https://pypi.org/project/pyupgrade is pretty useful for finding and fixing these things.

Reply:
https://github.com/getredash/redash/pull/4533